app/controllers/v1/nod_callbacks_controller.rb

```diff
@@ -80,7 +80,7 @@ def authenticity_error
     end
 
     def bearer_token_secret
-      Settings.dig(:nod_vanotify_status_callback, :bearer_token)
+      Settings.nod_vanotify_status_callback.bearer_token
```

Comment:
This setting exists in the manifest repo and `config/settings/test.yml`, but not settings.yml (i.e. same issue as `Settings.pension_ipf_vanotify_status_callback.bearer_token`).